### ISSUE TITLE
rounded values XLS import problem

### ISSUE THREAD

**Issue by charvotj** [ONLYOFFICE/DesktopEditors#789]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (RPM package)

### Version information

6.4.2.17

### Expected Behavior

Import data from XLS file correctly

### Actual Behavior

When opening an XLS file with very low values (eg. 0,00000001039792031), program seems to round them to a clear "0". If I then write the exact same values manualy and save file as XLSX, the right values remain.

### Additional information

It could be related to the czech decimal point ",", but other (shorter) decimal values remains correct.

I'm actually using Manjaro and I'm not sure if my version is the RPM or DEB package.

**Comment by ShockwaveNN** [comment]
Are you sure it's rounded, but not just displayed with low 

See this:
![image](https://user-images.githubusercontent.com/668524/146678734-6d78b70c-0a95-4b21-9d80-a38050ace0e1.png)

And after I clicked `Increase decimal` on top toolbar several times I got:
![image](https://user-images.githubusercontent.com/668524/146678750-73bd5a5f-e754-48af-8ec6-798f570184cc.png)

If this is not helpful - please attach a file with example and we'll analyze it

**Comment by charvotj** [comment]
Yes, I'm pretty sure, because there was also chart inserted and after importing it was empty (all values were zero).

Here is a screenshot, problem is in the collumns C and F:
![image](https://user-images.githubusercontent.com/96319354/146764658-28aba040-c539-4836-9c12-fefb9ab6ceb2.png)
[bad import file.xls](https://github.com/ONLYOFFICE/DesktopEditors/files/7745076/bad.import.file.xls)

I've attached that file so you can analyze it.

**Comment by alinasafonova** [comment]
Hello, @charvotj
I can confirm - this is bug, issue 54623 in our private issue tracker.
Thank you for report.

**Comment by ShockwaveNN** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too
